internal/scripts/lint.ts

```diff
@@ -9,7 +9,7 @@ async function main() {
 	try {
 		await exec(
 			'yarn',
-			['prettier', shouldFix ? '--write' : '--check', '--log-level=error', '--cache', relativeCwd],
+			['prettier', shouldFix ? '--write' : '--check', '--log-level=warn', '--cache', relativeCwd],
```

Comment:
we should leave this as an error methinks

Reply:
When prettier encounters a file it can't format, it won't output any information about that when the log level is error. Those are counted as warnings. It means that it fails silently without any feedback.